src/mon/OSDMonitor.cc

```diff
@@ -8491,88 +8490,73 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
       return prepare_set_flag(op, CEPH_OSDMAP_NODEEP_SCRUB);
     else if (key == "notieragent")
       return prepare_set_flag(op, CEPH_OSDMAP_NOTIERAGENT);
-    else if (key == "sortbitwise") {
+    else if (key == "sortbitwise" ||
+      key == "recovery_deletes" ||
+      key == "require_jewel_osds" ||
+      key == "require_kraken_osds") {
       if (!osdmap.get_num_up_osds() && sure != "--yes-i-really-mean-it") {
         ss << "Not advisable to continue since no OSDs are up. Pass "
-           << "--yes-i-really-mean-it if you really wish to continue.";
```

Comment:
You are removing a line that is properly aligned with its parent and replacing it with the basically the same line but no longer aligned.